### ISSUE TITLE
Anomaly: Hollow Asteroid

### ISSUE THREAD

**Issue by ViolentBeetle** [MoreEventsMod/More_Events_Mod_Beta#258]
Idea for a L2 anomaly spawned on Asteroids + an event chain. In addition to simple results, can also unlock an event chain. 

**Anomaly desciption**
This asteroid has mass too low for its enourmous size, suggesting that beneath the rock surface it might actually be hollow. 

**Success: Space Mausoleum**
As our science team found its way inside, they have discovered something unexpected: dead aliens. Over a hundred bodies are lined by the walls of the asteroids, dressed in ornamental clothes and accompanied with variety of objects, preserved by vacuum of space. Analysis suggests they all died over 300,000 years before. This structure is an invaluable (and the only) source of knowledge about a mysterious alien species, their ways of life and their physiology.

>Asteroid starts producing society research
>[Generic] We can learn a lot from this
>[Xenophiles] Their customs were fascinating
>[Spiritualists] They were seeking to be close to the divine
>[Materialist] Haven't they heard of recycling?

**Success: Space Resort**
Insides of the asteroid revealed an incredible sight. Multiple habitat capsules are clinging to the walls of the asteroid while its centre houses a massive transparent sphere containing multiple artificial environment. Long abandoned and deactivated, it can be reactivated with ease, and with minimal adjustment serve our own needs.

>[Not fanatic collectivist] Convert it into a commercial resort [Starts producing energy]

>[Not fanatic individualist] Ensure as many citizens as possible enjoy this place [Gain "Asteroid relaxation" empire modifier: +5% happiness for 5 years]

>Dismantle the structure and study it [Gain engineering]

**Success: An Art Exhibition**
Our scientists have found something very strange. Inner walls of the asteroids are dotted with thousands of large crystals covered in pictogram and geometrical patterns. Between different shapes, colours and carvings, the team could not find a single pair of identical crystals. Every single crystal is now imaged for the archives, but what their creator was trying to say might forever remain a mystery.

>Gain influence

>[Generic] Those a gorgeous
>[Xenophile] If only the creators were still around to share their vision with us
>[Spiritualist] A fitting tribute to the miracles of creation
>[Materialist] If nothing else, a good exercise in set theories

**Success: Panopticon**
When the science team breached the interior of the asteroid, they found countless small living pods fixed on the outer wall. Large sphere is fixed in the centre allowing clear view on every pod. It appears, the asteroid was used by an unknown to us ancient species as a high security prison, under the idea that a handful of guards could covertly observe the inmates and remains of both are still littering structures. From data traces pulled from the station's computers, it appears that a faulty software update has locked inhabitants out of the system, leaving them to starve. Although it's not clear why they weren't rescued or have their bodies recovered.

>We must study the prison's design [Asteroid starts producing engineering]

**Failure**
The asteroid was found to be indeed hollow. In fact, it's not a single asteroid per se, but a group of several smaller asteroids that somehow got stuck together and were welded together by aeons of diffusion. A curious structure, but of little value.

>Disappointing

**Extra event chain: Kraken**
As an alternative success, anomaly can start kraken chain. 

**Successful anomaly research**
As our science team has progressed inside the hollow asteroid, they have found a complex tunnel system, leading towards a hollow core. Clearly artificially designed, they are lined with cables, support pylons and large canisters of unknown purpose, sealed shut and resistant to opening. A small orb is concealed deep inside the asteroid, containing data terminals. It will take time to decode information they contain, however.

>Get onto it.
Issues a special physics project for a level 3 scientist to decode data terminals.

**Successful decoding: Kraken in Making**
The data we pulled from terminals inside [Asteroid] contains information that is both incredible and highly disturbing. This entire place was suppose to become a body of powerful being, referred to as Kraken, product of genetic engineering by an ancient spacefaring race. The core and the tunnels are to be filled with organic tissue grown from gene seed preserved inside canisters. Self-propelled, self-sustaning and under its creator's mental control, it would be able to take on entire fleets by itself and win.

It appears, however, that this project was unfinished. The age of canisters caused genetic damage to the Kraken gene seed making it unusable. However, with its incredible regenerative capacity, introduction of tissue from another Kraken could repair it and allow us creation of our own beast.

>Try finding another Kraken to harvest [Event chain continues]

>Scuttle the project. Some things are better left unborn [Gain influence and society research, asteroid is removed]

**Hunt for the Kraken**
Should the player choose to pursue the Kraken, new system (Proposed name Pontoppidan, but generic would do if needed) is spawned. It contains no habitable planets, some barren and toxic worlds with no resources and an asteroid belt. Inside an asteroid belt is placed Kraken.

Kraken is a special ship that uses an asteroid as its model. It has very tough armour and a lot of HP, regenerative hull, very weak engines as well as weapon and strike crafts hangars used by the Scourge. A single ship should probably have about 15-20K firepower. Nothing of its tech should be salvageable.

Once Kraken is defeated it's disabled, Society project for Level 4 scientist is spawned to harvest its gene seed. 

**On finishing the project**
Using drilling equipment, our scientists have successfully penetrated the skin of the beast and extracted a large quantity of its tissue. This should be enough to extract gene seed needed to jump-start our own Kraken project. As they departed, [Science Officer], worrying that the creature can still be alive, planted an explosive device into the wound, ensuring that the Kraken would be torn apart as they leave. We can now restart the Kraken project, all we need a volunteer with mind powerful enough to become one with the beast and bring it out of control who's willing to sacrifice self to give birth to a powerful beast. 

>We should start looking for suitable candidates

Level 5 society project "Use gene seed and a volunteer to give birth to a Kraken" is spawned on the original asteroid. 

**On finishing the project**
The final preparations are complete. With strands from another Kraken insert in gene seed pods and [Scientist] positioned himself in a control room the project began. Sizzling, bubbling biomass had flooded the tunnels and the core, solidifying and differentiating, giving birth to various systems of a powerful beast. We had soon lost contact with our volunteer, until we received sudden transmission from inside.

[Scientist] absorbed and transformed. He changed, his body destroyed, his mind transformed and turned one with the Kraken but his loyalties remained unchanged. The mighty beast is born and is awating our orders.

>It's alive!
>The scientist is killed and kraken ship is spawned under player's control. For the sake of balance, it should probably be less powerful. The chain ends.

**Comment by Elimdur** [comment]
RELEASE THE KRAKEN xD

Those are some excellent ideas.

But there is a minor thing about that " Radiological analysis". If you by that mean Radiocarbon dating, this would only work if you have a reference of ther usual living space and food source. As that only is a comparison of the amount of an isotope of carbon between the sample and its sorrounding living space and then a mathematical thing how long it takes for that reference amount to decrease to the value you found in the bodies or whatever ;) So without a reference you cannot date anything unfortunately.

**Comment by ViolentBeetle** [comment]
OP edited, now it's just Analysis. Nobody suspects a thing.

**Comment by ViolentBeetle** [comment]
**Extra critical failure**
It appears that the hollow part of the asteroid was hiding an alien installation of a sort. However, as our science team managed to breach the wall, they triggered some sort of self-destruction sequence vaporising the asteroid itself, alongside with them and their ship.

**Comment by ViolentBeetle** [comment]
**Extra chain: Pirates**
It appears that naturally occurring cavern system inside the asteroid has attracted a group of criminals, who did not take kindly to our probing of their hideout. They are scrambling a small fleet of armed ships, ready to destroy anyone who try and disrupt their operations. 
>Spawn a pirate fleet. Exact configuration of it I leave to someone with better understanding of balance. It should probably have some mid-tier tech worth salvaging.

**Pirate fleet defeated**
The last pirate defenders have been neutralized. A tactical team can now enter their hideout to take out the gang for good.

>Prepare for tactical insertion [Spawn "Board pirate hideout special project for troop transport]

>[Military ethos only] No need to play coy. Take the entire base out [Asteroid is destroyed, gain some minerals and influence, end the chain]

**Boarding the base: listening post**
Our tactical team had entered the hideout unopposed. It seems the base was not as populous as we anticipated with all personnel had been manning the raider crafts. What we found inside, however, was a considerable amount of sophisticated communication and data processing equipment that was used to spy on civilian traffic and provide surveillance data elsewhere. 

Only one of the following confirmations can be offered at the same time.
>Some of their navigational data seem useful [Fully survey two random systems]
>Their communication techniques are very interesting [Give 10% progress toward next sensor tech]
>[This one should have higher chance, for being more interesting]We should be able to trace the recipient of the intel [Progress towards "Pirate bay" path]
>This equipment can be used in civilian sector [Gain some energy]

**Boarding the base: Pirate hideout**
Our tactical team had made its way through pirate's defences encountering desperate, but fruitless resistance from the outlaws. Many of base defenders were killed, but some were successfully subdued unharmed and are now detained, awaiting interrogation. A respectable amount of loot they stole from civilian fleets over the years had been seized.

>[Generic response]Good job! [Gain minerals. Progress towards one of the two Pirate Trial events which fires only a couple of days later.]
>[Special Pacifist response] Shame it has to come to this.

**Pirate Trial: Our own**
This event will fire if player made no contact with a spacefaring empire, 50% chance otherwise

The pirate gang we have arrested was composed of citizens of our own empire! Those marginals had sneaked outside of our main travel routes on stolen ships and used those to make a living by smuggling and raiding civilian traffic. Further interrogations confirmed that the pirates were acting alone and now all are either dead or detained. We can assure our citizens that the terror is finally over.

>[Any democracy or oligarchy + pacifict autocracy] Hold a public trial for their crimes [Empire gains 10 years of "Protector of the people" modifier, +5 happiness. Any foreign empire known at the time of event gets +10 opinion "Pirate hunter" that is ticking down]
>[Any oligarchy or any autocracy but pacifist] Summarily execute the traitors [Gain influence, same "Pirate hunter" modifier]
>Offer them a deal [Progress further toward one of the deal events]

**Pirate trial: Foreign pirates**
This event can only fire if player knows at least one other empire

The pirate gang we have in our custody was found to be mostly composed of marginalized citizens from [EmpireName]. They have terrorised civilian shipping lanes from years, mostly targeting their former nation. Further interrogations made it clear that they were acting alone and now the entire gang is either dead or arrested.

[EmpireName] have submitted formal request for extradition of the pirates so they can stand trial for their crimes against their homeworld.

>[Not xenophobe] Extradite the pirates [+30 optinion modifier from the target empire "Extradited our most wanted", ticking down]
>Additionally all options from own pirates apply, but entail -20 opinion penalty for trial or deal "Refused extradition request" and -30 for execution "Executed our citizens"

**The Deal: Secret stash**
50% probability

We had stricken a bargain with the detained pirates. In exchange for leniency they agreed to provide us with locations of massive stash of loot they acquired over the years of operation. Among recovered objects we found quite a lot of valuable hardware and precious metals. 

>Gain mineral and energy

**The Deal: Raiding teactics**
35% probability

In exchange of reduced sentence, some of the pirates have offered us their tactical expertise. Years of raiding have taught them quite a lot about evasion and maneuver and our own fleet command has a lot to learn from them 

>Gain "Raiding tactics" empire modifier [+5% evasion, +10% faster emergency jump]

**The Deal: Hidden battlecruiser**
15% probability

Arrested pirates offered to direct us to something special they found. During one of their raids they have ran into an advanced warship, abandoned by its crew after it sustained major damage. They have towed it towards a nearby asteroid field and hidden in hope to one day be able to reactivate it, but this day have never came. But with resources of our empire, we should have no problem restoring the warship to working condition

Spawn special engineering project on an asteroid for L3 scientist to repair the ship. Upon success: 

The warship pirates led us to is indeed fascinating. Now as we patched it up and powered its systems, once abandoned by its original crew, the might vessel is ready to serve in our fleet.

>Bring it online [player gets a Privateer-class Battlecruiser, with endgame weapon and Fallen Empire skin, which however can not ever be upgraded]

**Pirate Bay**
Should player get option to track down the recipient, he gets a special physics project to do so. Upon completion:

We have managed to crack the encryption of pirate communications and although whoever was receiving the data quickly discovered us and remotely wiped our equipment on our side, we were able to pinpoint their location. We should send a military fleet to investigate.

>Spawn "Pirate Bay" system with several molten and toxic worlds and an asteroid field. Inside spawn 3 midgame-powered pirate fleets (Probably about 5K each, one fleet equipped with energy, one with kinetic and one with lasers) as well as 5K power pirate base. 

As we were expecting: The system is teeming with pirates. They used their considerable resources plundered from civilian fleets across the galaxy to build up quite a power base.

>Prepare to attack [Close the event and have ships turn hostile]

>Try to turn the pirates against each other
This option would summon a conversation with a pirate captian. Depending on how much energy does your empire have, his chances to become cooperative would increase with him asking for a random sum of energy below what you have to hire him. He can also simply request a pardon and a job in your empire. 

proposed mechanism: 
Events with equal probability are added to the pool once certain energy threshold is cleared, ie
100, 500, 1000, 2000 and 5000 with pirate captain would as for this sum. Additionally with the same base probability he would ask for simple pardon. Fourth times weight chance for him to refuse. So without energy, there's 20% he'll just ask for a pardon, with 100 energy there's 66% he'll refuse and 17% hell as for 100 energy and 17% to ask for 100 credits and so on.

**Uncooperative**
You are taking me for a fool? None of us will ever betray the pirate brotherhood to you! You better leave our place and never come back, if you value your precious life and your ships.

**Cooperative**
You want to make a deal? Perhaps we can come to an agreement. Outlaw life had became stale and doesn't pay quite as it used to be. Perhaps mercenary work would suit me better. For {a small fee of X energy credits and} a promise to let me join your ranks, I will not fight you. My friends here, however, might not be so receptive to the idea, but together we should be able to take them out in no time.

>If player makes a deal, one of the fleets joins the side of the empire (And instantly comes into fight with other two). It is led by an admiral with special trait "Space Pirate" which increases his evasion, speed and emergency FTL buildup. His fleet is filled with pirate ships, corvette and cruiser style that can not be upgraded, using mid-game tech.

**Once battle is won and the pirates are taken out**
As the last outlaw ship was destroyed we can rejoice in knowing we have eradicated this gang. Commercial ships across the galaxy can be at ease, knowing that the menace had been eliminated. Considerable resources were recovered from the wreckage of the pirate station.

>[Generic response] We made the galaxy a safer place [Gain minerals, energy]
>[Pacifist response] Shame we had to resort to this

**Comment by Elimdur** [comment]
The small anomalies are already done. If we go for are release soon we can just exclude the bigger ones for now. I´ll just go on working with the rest of the events for the next release then.

**Comment by Elimdur** [comment]
Still the Pirate system and kraken quest line to do
